sapling-dev-server/userSaplings

```diff
@@ -1 +1,14 @@
-[]
+[
+  {
+    "displayName": "Circuits",
+    "namespace": "circuits",
+    "runtimeFiles": [
+      "localhost:3030/sapling-dev-server/circuits/js/circuits.js"
+    ],
+    "styleFiles": [
+      "localhost:3030/sapling-dev-server/circuits/css/circuits.css"
+    ],
+    "workerFiles": [],
+    "icon": "https://via.placeholder.com/48/333333/FFFFFF?text=C"
```

Comment:
I think we should skip the placeholder. My browser is blocking it now too. Davey might have an icon we can use instead. I have a PR up in grid that shows how to include an icon in the sapling dev server.

Reply:
Addressed